site/dat/repo/jpgc-plugins.json

```diff
@@ -748,187 +748,6 @@
       "com.googlecode.jmeter.plugins.webdriver.sampler.gui.WebDriverSamplerGui"
     ],
     "versions": {
-      "1.4.0": {
```

Comment:
According to usage statistics, there's still many people using older versions, thus these versions should be still offered.  The only versions I we might delete are those that are old _and_ have less than 1% of usage. There are just couple of those.